kalliope/tts/watson/watson.py

```diff
@@ -70,5 +73,5 @@ def _generate_audio_file(self):
 
     def get_payload(self):
         return {
-            "text": self.words
+            "text": "<s><prosody pitch='"+self.pitch+"' rate='"+self.rate+"'>" + self.words + "</prosody></s>"
```

Comment:
Hi, could you please give a pointer to the doc that deals with this payload syntax? Thanks.

Reply:
Hi! Here are the docs I reviewed looking for use that functionality.

[Watson supports SSML](https://cloud.ibm.com/docs/text-to-speech?topic=text-to-speech-ssml). Also, the tag <s> is for sentence, explained [here](https://www.ibm.com/support/knowledgecenter/SSMQSV_6.1.1/com.ibm.voicetools.ssml.doc/ssml_tags.html).